calendar/calendar.widget/index.coffee

```diff
@@ -64,12 +68,23 @@ updateHeader: (rows, table) ->
     tableRow.append "<td>#{dayOfWeek}</td>"
 
 updateBody: (rows, table) ->
+  #Set to 1 to enable previous and next month dates, 0 to disable
+  PrevAndNext = 1
```

Comment:
Probably good to move this to the top with all the other configuration option. JS also has true booleans so you can use 'true' and 'false' instead of  0 and 1.

Finally, it is also a convention to have variable names starting in lowercase (not a big issue of course)